src/utils/index.tsx

```diff
@@ -57,7 +57,10 @@ export function getCookieValue(
         .map((c) => c.substring(name.length + 1))[0] ?? null
 }
 
-const PathInfo = t.type({ typename: t.string, currentPath: t.string })
+const PathInfo = t.intersection([
+  t.type({ typename: t.string, currentPath: t.string }),
+  t.partial({ instance: t.string }),
+])
```

Comment:
Could you explain the logic behind this?

Reply:
Yes, let's use the call later :smile: